lib/thinreports/generator/pdf/document/page.rb

```diff
@@ -26,6 +26,16 @@ def start_new_page(format)
           stamp(format_id.to_s)
         end
 
+        def start_new_page_for_report(format)
+          @current_page_format = format
+          pdf.start_new_page(new_basic_page_options(current_page_format))
+          pdf.move_down current_page_format.page_margin_top
+        end
+
+        def max_content_height
+          pdf.bounds.height - current_page_format.page_margin_top - current_page_format.page_margin_bottom
```

Comment:
この `pdf.bounds` は prawn 上の現在ページの bounds を返すってことなんだろうか。

> The bounds method returns the current bounding box you are currently in, which is by default the box represented by the margin box on the document itself. When called from within a created bounding_box block, the box defined by that call will be returned instead of the document margin box.

Reply:
現在のbounding boxの領域を返すみたいですね。
bounding box を変更するとページサイズではなくなってしまうので、`pdf.margin_box` とかの方が良いのかもしれない

Reply:
margin_box が正しい気がしますね。 `start_new_page` で定義されてその後変更されないのでページのサイズを取るならこちらが正しいと思う。
https://github.com/prawnpdf/prawn/blob/c77e859ea58bd3c7e553e01abcc343e3948c4c6b/lib/prawn/document.rb#L691

Reply:
https://github.com/standfirm/thinreports-generator/pull/6/commits/0b186f0f0f847b1b277e8c947effd2795cd7a49f で対応しました。